README.md

```diff
@@ -1,9 +1,15 @@
 
 # Kubernetes Provider for Terraform [![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/hashicorp/terraform-provider-kubernetes?label=release)](https://github.com/hashicorp/terraform-provider-kubernetes/releases) [![license](https://img.shields.io/github/license/hashicorp/terraform-provider-kubernetes.svg)]()
 
-<a href="https://terraform.io">
-    <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" alt="Terraform logo" title="Terrafpr," align="right" height="50" />
-</a>
+This repo is a fork of the terraform-provider-kubernetes. It only exists during
+the time required to review our feature. Ti publish the provider, we are using
+the
+[GoReleaser](https://www.terraform.io/docs/registry/providers/publishing.html#using-goreleaser-locally)
+to avoid any change in the upstream github workflows.
+
+<a href="https://terraform.io"> <img
+src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg"
+alt="Terraform logo" title="Terrafpr," align="right" height="50" /> </a>
```

Comment:
I feel like this part of the diff could be avoided. It might make it easier to track upstream in the future if we minimize changes.